### PR TITLE
xe-install-supplemental-pack: use update-upload

### PR DESCRIPTION
**Pull request by srowe**

### CODE REVIEW

File: scripts/xe-install-supplemental-pack [hunk 1]
```diff
@@ -29,13 +31,6 @@ done
...
-  pool_uuid=$(xe pool-list --minimal)
-  sr_uuid=$(xe pool-param-get uuid=$pool_uuid param-name=default-SR --minimal)
-fi
-
-vdi_uuid=$(xe vdi-create sr-uuid=$sr_uuid name-label="$(basename $iso)" virtual-size=0)
-trap "xe vdi-destroy uuid=$vdi_uuid" EXIT
-xe vdi-import uuid=$vdi_uuid filename=$1
-update_uuid=$(xe update-introduce vdi-uuid=$vdi_uuid)
-xe update-apply uuid=$update_uuid
+update_uuid=$(xe update-upload file-name="$iso" ${sr_uuid:+sr-uuid=$sr_uuid})
```
Comment (lindig) [inline scripts/xe-install-supplemental-pack:34]: I'd recommend `${sr_uuid:+sr-uuid="$sr_uuid"}`.

Comment (srowe) [reply scripts/xe-install-supplemental-pack:34]: If we get spaces in UUIDs we're in a bit of trouble.

**Comment by jonludlam** [comment]
@srowe: could you add a signed-off-by line and a description to the PR please?

**Comment by jonludlam** [comment]
Actually given that I can 'rebase and merge' without creating a merge commit with the PR description in, this'll do.

**Pull request merged**